js/main.js

```diff
@@ -25,8 +25,12 @@
                 id = currentPhoto.id;
                 title = currentPhoto.title;
                 secret = currentPhoto.secret;
-                picUrl = "https://farm"+farm+".staticflickr.com/"+server+"/"+id+"_"+secret+"_z.jpg";
-                $('.image-slider-wrapper ul').append("<li class='active imgArray' ><a data-lightbox='image-set'" + " href='" + picUrl + "'><img class='active' "+ " src='" +picUrl+"'/></a></li>");
+                picUrl = "https://farm" + farm + ".staticflickr.com/" + server + "/" + id + "_" + secret + "_z.jpg";
+
+                if(i%4 === 0){
+                    $('.image-slider-wrapper').append("<div class='row'></div>");
+                }
+                $('.image-slider-wrapper>.row:last-child').append("<a data-lightbox='image-set'" + " href='" + picUrl + "'><img class='col-xs-3' "+ " src='" +picUrl+"'/></a>");
```

Comment:
Noticed here that you have `class="col-*"` on the <img /> tag. The column and row classes are meant for layout, rather than UI elements. I see that you're trying to have a responsive grid of images. The best solution would be to construct the wrapping `<div />` with the appropriate row/column class, and applying the bootstrap `img-responsive` class.

The final HTML out should be similar to:
```html
<div row>
  <div col-xs-3>
    <img img-responsive />
  </div col-xs-3>

  <div col-xs-3>
    <img img-responsive />
  </div col-xs-3>

  <div col-xs-3>
    <img img-responsive />
  </div col-xs-3>

  <div col-xs-3>
    <img img-responsive />
  </div col-xs-3>
  ... 
</div row>
```
http://getbootstrap.com/css/#images

That way, as the columns flex to meet the browser width, the images resize appropriately. Let me know if you have any questions.